avocado/utils/cpu.py

```diff
@@ -114,3 +114,12 @@ def get_cpu_arch():
         return 'x86_64'
     else:
         return 'i386'
+
+
+def count_cpus():
```

Comment:
The Python standard library has `multiprocessing.cpu_count()` which can be used here.